### DISCUSSION TITLE
Unable to pick a directory?

### DISCUSSION THREAD

**Discussion by kazum1kun** [TheLastGimbus/GooglePhotosTakeoutHelper#275] [I NEED HELP] [answered]
Hello,
This might sound dumb but I'm stuck at the step where you pick the Takeout directory. 
There is no pop-up for directory, and I tried to type in manually - either way I can't progress further.
![image](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/4069734/bc9ae0ce-f03e-4565-bf66-930c329dd9f8)

I tried running the program directly, in PowerShell 7, or a cmd - none of them works.

I have Windows 23H2 (22631.2861), if that matters.

Thanks!

**Comment by TheLastGimbus** [comment, marked as answer]
Try clicking stuff back and forth, both keyboard and mouse

i know this doesn't work very well... i consider making it a nice UI app instead of pure cmd in future

**Reply by kazum1kun** [reply]
oh wow thanks, I entered some random stuff and it did show up.

**Reply by TheLastGimbus** [reply]
See that's some professional programming right there :sunglasses:
